src/pages/videos/videos.scss

```diff
@@ -0,0 +1,45 @@
+page-videos {
+  ion-grid {
```

Comment:
All css selector (unless video-container) are useless on this page, bad copy/paste